### ISSUE TITLE
False positive for unused-import on class keyword arguments

### ISSUE THREAD

**Issue by scop** [PyCQA/pylint#3202]
### Steps to reproduce
```
$ cat t.py
from const import DOMAIN

class Child:
    def __init_subclass__(cls, domain, **kwargs):
        pass

class Parent(Child, domain=DOMAIN):
    pass
$ cat const.py 
DOMAIN = "meh"
```

### Current behavior
```
$ pylint t.py | grep unused-import
t.py:1:0: W0611: Unused DOMAIN imported from const (unused-import)
```

### Expected behavior

No unused-import warning.

### pylint --version output

2.4.3
